kits/java/simple/lux/Unit.java

```diff
@@ -49,4 +49,4 @@ public String buildCity() {
   public String pillage() {
     return String.format("p %s", this.id);
   }
-}
\ No newline at end of file
+}}
```

Comment:
extra brace here?

Reply:
this would definitely fail our tests (recommend u run `npm test` first to verify things generally work)

Reply:
Thanks.  I wasn't sure how to have the system proof-read my code.